### PR TITLE
fix project delete api

### PR DESCRIPTION
**Pull request by BehnamAxo**

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -122,6 +122,10 @@ function handleRestRequest (requestType, requestURI, requestData, callback) {
 				request_options.data = requestData;
 			}
 
+			if('del' === requestType && requestData){
```
Comment (jdgarcia) [inline lib/api.js]: you can just combine this `if` with the one above.
`if (requestData && ('get' === requestType || 'del' === requestType)) {`

**Comment by BehnamAxo** [comment]
@jdgarcia done

File: lib/projects.js [hunk 2]
```diff
@@ -95,12 +95,13 @@ var Projects = {
 	 * Deletes a project
 	 *
 	 * @param		integer		project_id		ID of single project
+	 * @param 	integer   project_to_move_to
```
Comment (jdgarcia) [inline lib/projects.js]: can you also add a description for this parameter?

Comment (BehnamAxo) [inline lib/projects.js]: @jgrosso yes sure

Comment (BehnamAxo) [reply lib/projects.js]: @jdgarcia done

**Pull request merged**